src/app/code/community/Omise/Gateway/Block/Form/Offsiteinternetbankingpayment.php

```diff
@@ -11,7 +11,7 @@ class Omise_Gateway_Block_Form_Offsiteinternetbankingpayment extends Mage_Paymen
     protected function _construct()
     {
         parent::_construct();
-        $this->setTemplate('payment/form/omiseoffsiteinternetbankingpayment.phtml');
+        $this->setTemplate('payment/form/omise/omiseoffsiteinternetbankingpayment.phtml');
```

Comment:
is that part of another PR? that code?
you probably did not checkout from `master` this branch?

Reply:
I did, but another PR is incoming (waiting approval) that has that. My local project will not work without it so I needed it here too. It will all come out in the wash

Reply:
After bit thinking if it is since 5.4 I think we should add than we should keep:
`<?php echo "text" ?>`
Because we have no idea.. what kind of servers are used by customers.
And since it is Magento 1, I think better to play it safe.
And additionally I have checked magento 1.9.2.4 and there is no single place where they use construction `<?= ?>`